### ISSUE TITLE
Antialiasing gamma & TWDSS Visual shifting

### ISSUE THREAD

**Issue by Ronln** [fholger/vrperfkit#8]
Asgard's Wrath when Epic or High AA used the left eye getting a very bright gamma. It happens only after starting the game, not on the menu.
TWDSS on the bottom part of the screen is Visual shifting can be noticeable even in the menu.

**Comment by Ronln** [comment]
Ragnarock has the same issue as TWDSS it's like there is a circle and you see its edge and it's always vibrating especially if you look at the text through it.  Sorry for my noob's description :D

**Comment by fholger** [comment]
Yes, I'm seeing it, too, in TWD. I have no idea what's causing it, it makes absolutely no sense. :(
